modules/installation-osp-enabling-kuryr.adoc

```diff
@@ -47,4 +47,4 @@ platform:
 <1> The installation program automatically discovers both `trunkSupport` and `octaviaSupport`, so you are not required to provide values for these parameters.
 
 +
-At deployment, the Kuryr SDN is used in place of the OpenShift SDN.
+At deployment, the Kuryr SDN is used in place of the {product-title} SDN.
```

Comment:
@jboxman, is "OpenShift SDN" the correct terminology?

Reply:
He confirmed it in Slack - revert to "OpenShift SND."

Reply:
Answer is :+1: